contents/data_vis_makie_recipe.md

```diff
@@ -0,0 +1,104 @@
+## A Makie recipe for a DataFrame {#sec:recipe_df}
+
+Unlike other libraries that already support a wide set of input formats via recipes, i.e. `Plots.jl`, in `Makie.jl` most of the time we need to pass the raw data to functions.
+However, we can also define our own  `recipe` in  `Makie.jl`.
+A `recipe` is your own custom plotting type command.
```

Comment:
I still don't think this is clear.
For instance, a new user (the profile of our reader) would be confused.

Does a recipe create a `plot` and `plot!` custom function for my custom types?
Like call i simply call:

```julia
plot(my_custom_type)
```

and it will plot?

It is not clear.
Down below you create a recipe for a `DfPlot`:

```julia
@recipe(DfPlot, df) do scene
...
```

But somehow you have to call it with `dfplot` (all lowercase).
Our reader would be asking themselves "WTF, how does that work? Do I have to use `plot[!]`, or `dfplot`? And how did it got converted to a lowercase plotting function?"

Reply:
agree. Now it saids that the macro `@recipe` is responsible for creating two new lowercase functions that you are suppose to use later on.